configs/config.novacustom_nv4x

```diff
@@ -0,0 +1,34 @@
+CONFIG_LOCALVERSION="1.2.0"
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_TSEG_STAGE_CACHE=y
+CONFIG_VENDOR_NOVACUSTOM=y
+CONFIG_MAINBOARD_SMBIOS_MANUFACTURER="Notebook"
+CONFIG_BOARD_NOVACUSTOM_NV4X=y
+CONFIG_PCIEXP_HOTPLUG_BUSES=42
+CONFIG_PCIEXP_HOTPLUG_MEM=0xc200000
+CONFIG_PCIEXP_HOTPLUG_PREFETCH_MEM=0x1c00000
+CONFIG_USE_PM_ACPI_TIMER=y
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x30000
+CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="NV4XMB,ME,MZ"
+CONFIG_UART_PCI_ADDR=0x0
+CONFIG_PCIEXP_HOTPLUG_PREFETCH_MEM_BELOW_4G=y
+CONFIG_PCIEXP_HOTPLUG_IO=0x2000
+CONFIG_SUBSYSTEM_VENDOR_ID=0x0000
+CONFIG_SUBSYSTEM_DEVICE_ID=0x0000
+CONFIG_SMMSTORE_V2=y
+CONFIG_SMMSTORE_SIZE=0x40000
+# CONFIG_TPM_PPI is not set
+CONFIG_CBFS_MCACHE_RW_PERCENTAGE=50
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0=y
+CONFIG_POST_IO_PORT=0x80
+CONFIG_PAYLOAD_TIANOCORE=y
+CONFIG_TIANOCORE_CUSTOM=y
+CONFIG_TIANOCORE_REPOSITORY="https://github.com/Dasharo/edk2.git"
+CONFIG_TIANOCORE_UEFIPAYLOAD=y
+CONFIG_TIANOCORE_TAG_OR_REV="4d2846baa98c253eadfb4c4e3085138e0d1396ab"
```

Comment:
This revision has TPM PPI support: cad237255be0eed3b184e927871791057d8da9db

Reply:
Done

Reply:
Seems to work fine with this change. There's a `ppi` entry in `/sys/class/tpm/tpm0` now